### DISCUSSION TITLE
cloudflare做解析时开启proxy就无法访问国内的主机

### DISCUSSION THREAD

**Discussion by neozhu** [XIU2/CloudflareSpeedTest#344] [讨论]
![image](https://user-images.githubusercontent.com/1549611/137609720-695c1477-8cfc-472a-a9ab-b9684526eaa5.png)
比如我的域名，之前默认都是开始proxy 解析也是正常的，
今天突然不行了
![image](https://user-images.githubusercontent.com/1549611/137609735-1accd199-4571-41b9-8ec8-60d3da56312e.png)
只有关闭proxy 才能正常访问

难道是长城改了策略

**Comment by XIU2** [comment]
没明白你说的什么意思。

**Comment by neozhu** [comment]
开启 proxy,就无法打开 razor.i247365.net  IP不对
关闭 proxy DNS only  就可以打开

**Comment by XIU2** [comment]
打不开提示什么错误？

**Comment by neozhu** [comment]
![image](https://user-images.githubusercontent.com/1549611/137610678-58166412-1491-4a27-84f7-14e3fb573b1e.png)
开启就错误
关闭就正常了

**Comment by XIU2** [comment]
522 是 Cloudflare 链接源站时出现了问题。
没见到有人国内服务器套 Cloudflare 的。。。

**Comment by neozhu** [comment]
好吧，看来还是用阿里解析吧

**Comment by neozhu** [comment]
你们的有问题吗，好像就今天出了问题

**Comment by neozhu** [comment]
会不会是阿里云屏蔽了cloudflare的ip
